### DISCUSSION TITLE
Where i can get more info about Union store?

### DISCUSSION THREAD

**Discussion by reshke** [apache/cloudberry#538] [Q&A]
This commit mentions UnionStore https://github.com/cloudberrydb/cloudberrydb/commit/6dbd635e7a657d74ab8cc90bafcd106a4591b8db
However, design doc is not publicly open, neither I can see any documentation in open source. 
Is it Neon-like compute/storage separation solution?

**Comment by tuhaihe** [comment]
Hi @reshke, 

AFAIK, UnionStore is developed following the concept "Log is the Database" to build the unified storage for many workloads such as auditing, BI apps, and more. And it's used in a Cloud-Native(Compute/Storage decouple) arch.

Here is one Chinese blog[1] for UnionStore, welcome to take a reference. But it is still not in the open-source plan yet.

If I'm wrong, please correct me. @my-ship-it 

[1] https://blog.csdn.net/m0_54979897/article/details/133172314
